<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>706</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>58</y>
      <width>726</width>
      <height>409</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>610</y>
      <width>137</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(18, 183, 245);
    color: rgb(255, 255, 255);
    border-radius: 8px;
}

QPushButton:pressed {
    background-color: darkblue;
}

QPushButton:hover {
    background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>发送(S)</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>-66</y>
      <width>1030</width>
      <height>785</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/bg.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>610</y>
      <width>131</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(227, 227, 227);
    color: rgb(255, 255, 255);
    border-radius: 8px;
}

QPushButton:pressed {
    background-color: darkblue;
}

QPushButton:hover {
    background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>关闭(C)</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>522</y>
      <width>728</width>
      <height>147</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>textBrowser</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButtonClose</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
